examples/cdn-assets/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  assetPrefix: 'http://localhost:9999/'
```

Comment:
Add a comment saying something similar:

"In a real app, you need to put the CDN's URL prefix here"